env.sh

```diff
@@ -13,7 +13,26 @@ if [ -f $ENV_FILE ]; then
   echo "env file $ENV_FILE exists."
 else
   echo "env file $ENV_FILE does not exist. Using provided defaults"
-  cp .env.example .env
+  cp .env.example $ENV_FILE
+
+  if [ -z ${APOLLO_KEY} ]; then
+    echo "Missing Apollo Key Env"
+  else
+    echo APOLLO_KEY=$APOLLO_KEY >> $ENV_FILE
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/Wyvarn/space-xplorer-client/pullRequest?prid=6084143)